0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -123,6 +124,11 @@ Unsubscribe from ledger events.
 
 Query whether the plugin is currently connected.
 
+#### isOpen
+<code>ledgerPlugin.isOpen() ⇒ Boolean</code>
+
+Query whether the plugin is currently connected and has an active websocket.
```

Comment:
Gotcha.

> Perhaps the names should be changed to reduce confusion? s/isConnected/isReady and s/isOpen/isConnected, or similar.

That would make most sense to me. It would be annoying to have to make that breaking change, and to have yet another method on this interface.

What if `isConnected` tells you whether it's connected right now and the methods like `getPrefix` reject if the plugin hasn't ever been connected? You wouldn't be able to check beforehand if the plugin has ever connected, but you would probably just call `plugin.connect().then(() => plugin.getPrefix())` anyway.